src/styles/partials/_theme.scss

```diff
@@ -0,0 +1,169 @@
+body{
+	color: $primary-color;
+	margin: 0;
+	padding: 0;
+	width: 100vw;
+	min-height: 100vh;
+	background: #fff;
+	overflow-x: hidden;
+}
+
+a {
+	color: $link-color;
+}
+
+a:hover {
+	color: $link-hover-color;
+	text-decoration: underline;
+	cursor: pointer;
+}
+
+hr {
+	margin-top: 20px;
+	margin-bottom: 20px;
+	border: 0;
+	border-top: 1px solid $line-color;
+}
+
+
+
+/****HOME PAGE SEARCH BOX ELEMENTS****/
+
+
+.input-group {
+	display: table;
+}
+
+.input-group-btn {
+	display: table-cell;
+	width: 1%;
+}
+
+.btn {
+	border: 1px solid transparent;
+	display: inline-block;
+	padding: 6px 12px;
+	vertical-align: middle;
+	border-radius: 4px;
+	border-top-left-radius: 0px;
+	border-bottom-left-radius: 0px;
+}
+
+/****NAVBAR ELEMENTS****/
+
+.navbar {
+	background-color: $background-color-navbar;
+	border:1px solid $border-color-navbar;
+	margin: 0;
+	padding: 0;
+}
+
+	#navbar-toggle-cbox {
+		display: none;
+	}
+
+	.navbar-header {
+		width: 100%;
+	}
+
+		.navbar-brand {
+			display: block;
+			height: 50px;
+		}
+
+		.navbar-toggle {
+			border: 1px solid $border-color-button;
+			border-radius : 4px;
+			float: right;
+			margin: 8px 15px 8px 0px;
+			padding: 7px 8px;
+		}
+
+	.navbar-collapse {
+		display: none;
+		border: 1px solid $border-color-navbar;
+
+		li {
+			position: relative;
+			display: block;
+
+			a {
+				color: $navigation-link-color;
+				display: block;
+				padding: 10px 15px;
+				line-height: 20px;
+			}
+
+			a:hover {
+				color: $primary-color;
+				text-decoration: none;
+			}
+
+		}
+	}
+
+		#navbar-toggle-cbox:checked ~ .navbar-collapse {
+		  display: block;
+		}
+
+@media #{$breakpoint-small} {
+	.navbar-header {
+		width: auto;
+		float: left;
+	}
+
+	.navbar-toggle {
+		display: none;
+	}
+
+	.navbar-collapse {
+		display: block;
+		height: 50px;
+		border: none;
+
+		ul {
+			position: relative;
+			float: right;
+
+			li {
+				float:left;
+
+				a {
+					padding-top: 15px;
+					padding-bottom: 15px;
+				}
+			}
+		}
+	}
+}
+
+/****ICONS****/
```

Comment:
No need of using custom SVGs you can simply use Material icons which are already being included, in the project.